### PR TITLE
theme-color-quick-fix

### PR DESCRIPTION
**Pull request by jongbonga**
This fixes the primary contrast's color which was previously defaulted to #000000, it also updates tailwind colors link

### CODE REVIEW

File: web-client/src/theme/variables.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@
   // UI Kit: Primary 2 - Portland Orange
   --ion-color-primary: #ff642e;
   --ion-color-primary-rgb: 255, 100, 46;
-  --ion-color-primary-contrast: #000000;
+  --ion-color-primary-contrast: #ffffff;
```
Comment (PiDelport) [inline web-client/src/theme/variables.scss:35]: Is there a reason for deviating from the Ionic color generator's contrast color here?

If so, we should document it?

File: web-client/src/theme/variables.scss [hunk 2]
```diff
@@ -32,7 +32,7 @@
...
   --ion-color-primary-contrast-rgb: 0, 0, 0;
```
Comment (PiDelport) [inline web-client/src/theme/variables.scss:36]: (The `contrast` and `contrast_rgb` variations should match.)

**Review by PiDelport — CHANGES_REQUESTED** [review]

File: web-client/src/theme/variables.scss (hunk 1 above)
Comment (longtomjr) [reply web-client/src/theme/variables.scss:35]: This contrast match the designs, so yep, we should document it for now. Also not sure what the contrast ratio is here, if it is not great we should raise this with Abby

File: web-client/src/theme/variables.scss (hunk 2 above)
Comment (jongbonga) [reply web-client/src/theme/variables.scss]: Done! Thanks

**Review by PiDelport — APPROVED** [review]
Looks good! 🚀

**Pull request merged**